addons/mail/wizard/email_template_preview.py

```diff
@@ -38,8 +38,10 @@ def default_get(self, fields):
     @api.onchange('res_id')
     @api.multi
     def on_change_res_id(self):
+        if not self.res_id:
+            return {}
         mail_values = {}
-        if self.res_id and self._context.get('template_id'):
+        if self._context.get('template_id'):
```

Comment:
Here we decided to update the body value only if user select a actual record from selection, if user select blank value then simply don't update anything.Thank